pkg/monitor/monitor/util.go

```diff
@@ -614,12 +654,18 @@ func getMonitorVolumes(config *core.ConfigMap, monitor *v1alpha1.TidbMonitor, tc
 	}
 	volumes = append(volumes, prometheusRules)
 	if tc.IsTLSClusterEnabled() {
```

Comment:
For most of the functions in the monitor module, they are built with the first TC in the `spec.clusters`, what if the clusters have different configurations or different versions?